lib/test/unit/autorunner.rb

```diff
@@ -515,6 +519,15 @@ def load_config(file)
         @runner_options = @runner_options.merge(runner_options)
       end
 
+      def load_plaintext_config(file)
+        require "shellwords"
+        File.readlines(file, chomp: true).each do |option|
```

Comment:
We can use `line` for this block parameter because it's not option yet. (It becomes zero or more options after `Shellwords.shellsplit`.)

```suggestion
        File.readlines(file, chomp: true).each do |line|
```

Reply:
Sounds nice!